README.md

```diff
@@ -1,2 +1,31 @@
-# cloud-first-data-science-community
-GitHub home of cloud first data science community content
+# Cloud First Data Science Community Meetup
```

Comment:
Can we call this doc "meetup-landing-page.md" and leave the readme as is for now?

Reply:
Sure